lang/jp/README-jp.adoc

```diff
@@ -0,0 +1,299 @@
+// Copyright 2019-2022 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+= Vulkan^®^ Guide
+:regtitle: pass:q,r[^®^]
+The Khronos{regtitle} Vulkan Working Group
+:data-uri:
+:icons: font
+:max-width: 100%
+:numbered:
+:source-highlighter: rouge
+:rouge-style: github
+
+image::../images/vulkan_logo.png[Vulkan Logo]
```

Comment:
the image links are going to need to be adjusted

Reply:
Thanks for the catch Spencer!  I think these were the last two that were not adjusted correctly as the README-jp.adoc isn't processed by adoctor when building the guide (i.e. the rest spit out errors).
I'll resolve the conversation for now, please let me know if you find anything else.
I don't know if you're able but Marty has us scheduled to discuss the guide in the samples meeting tomorrow.